src/components/header/NavigationAvatar.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Box, HStack } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through `'@chakra-ui/react'`? Thanks